packages/nimble-components/src/wafer-map/specs/features/die-margins.md

```diff
@@ -0,0 +1,42 @@
+# Nimble Wafer Map Die Margins
```

Comment:
Hey @munteannatan, I'm very sorry, I had it mixed up in my head and should have double-checked. You're initial PR was correct in that the padding fills space inside the border while margin is the space outside the border between elements. https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Introduction_to_the_CSS_box_model#padding_area

Sorry about that 😅 but if you update the document to refer to padding again then we would be good to go.

Reply:
no problem, I should have double checked mysefl